config/application.yml.default

```diff
@@ -58,6 +58,7 @@ deliver_mail_async: 'false'
 deleted_user_accounts_report_configs: '[]'
 disable_email_sending: 'true'
 disallow_all_web_crawlers: 'true'
+disposable_email_services: '["courtsidemail.com","eyamail.com","fileright.net","immigrationdirect.com","kingfishmail.com","speedstermail.com","sprintpostmail.com","usimmigrationemail.com","firemailbox.club","safemail.icu","temp2-login.com","temp5-login.com","temp9-login.com","temp13-login.com"]'
```

Comment:
[To @pauldoomgov's point](https://github.com/18F/identity-idp/pull/5754#pullrequestreview-840547122), should we consider to empty this out in the default configuration and keep it only in the live production configuration?

Reply:
https://github.com/18F/identity-idp/pull/5773